### ISSUE TITLE
Contribute Glowroot 0.14-compliant Helm Chart to the Glowroot project

### ISSUE THREAD

**Issue by robinroos** [novumrgi/helm#20]
After #16 #18 #19 please consider perhaps contributing the Glowroot helm chart to the Glowroot project for ongoing maintenance.

**Comment by luhahn** [comment]
Yeah, i will talk to trask, maybe he is interested in hosting the helm chart :)
